torch/fx/experimental/proxy_tensor.py

```diff
@@ -211,9 +214,9 @@ def get_constant(idx):
 def maybe_disable_fake_tensor_mode():
     # TODO: figure out if this API generally makes sense and bake it into the
     # library
-    mb_fake_mode = _get_current_dispatch_mode()
+    mb_fake_mode = torch._C._get_fake_tensor_mode()
     if isinstance(mb_fake_mode, FakeTensorMode):
-        return _pop_mode_temporarily()
+        return unset_fake_temporarily()
```

Comment:
What does unset_fake_temporarily do if there is no mode? If it no-ops, you can just return this unconditionally

Reply:
fair